packages/react-imask/index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "react-imask" {
+  import { Component, InputHTMLAttributes, Ref } from "react";
+
+  export type MaskType = string | typeof RegExp | Date | Number;
+
+  /**
+   * For more info see the documentation: https://imask.js.org/guide.html#getting-started
+   */
+  export interface IMaskInputProps extends InputHTMLAttributes<HTMLInputElement> {
```

Comment:
I'm getting errors here since `max` and `min` are typed differently for `react-imask`.

Consider:

```typescript
type InputWithoutMaxMin = Omit<InputHTMLAttributes<HTMLInputElement>, "max" | "min">;
/**
 * For more info see the documentation: https://imask.js.org/guide.html#getting-started
 */
export interface IMaskInputProps extends InputWithoutMaxMin {
```